packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleTableTests.cs

```diff
@@ -86,6 +86,7 @@ public void NimbleTable_GivenUnsupportedData_ThrowsJsonException()
     {
         var context = new TestContext();
         context.JSInterop.Mode = JSRuntimeMode.Loose;
+        var table = context.RenderComponent<NimbleTable<TTable>>(p => p.Add(propertyGetter, propertyValue));
```

Comment:
Is this right? It looks like the variable is unused and the code is duplicate of the next line.

Reply:
Oops...I left in some code I thought I was going to change to but didn't.

Reply:
Fixed.